src/PowerShellEditorServices/Server/PsesDebugServer.cs

```diff
@@ -16,50 +16,72 @@
 
 namespace Microsoft.PowerShell.EditorServices.Server
 {
-    public class PsesDebugServer : IDisposable
+    /// <summary>
+    /// Server for hosting debug sessions.
+    /// </summary>
+    internal class PsesDebugServer : IDisposable
     {
-        protected readonly ILoggerFactory _loggerFactory;
+        private static bool s_hasRunPsrlStaticCtor = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this initialization to 's_hasRunPsrlStaticCtor', the compiler will do that for you.](https://app.codacy.com/app/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4579423)